topics/low-code/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: low-code-development, low-code-production
```

Comment:
Doesn't look like anyone has actually used `low-code-production` as a topic yet: https://github.com/topics/low-code-production. Let's leave it off.

```suggestion
aliases: low-code-development
```